solr/core/src/test/org/apache/solr/schema/BooleanFieldTest.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.solr.schema;
 
+import java.util.Map;
 import org.apache.solr.SolrTestCaseJ4;
 import org.junit.BeforeClass;
 import org.junit.Test;
```

Comment:
I think you could change this test to use solrconfig-tlog.xml instead of solrconfig-basic.xml. That way you could remove the change in this PR to solrconfig-basic.xml and keep that one simple.

Reply:
Ah didn't know there's a solrconfig-tlog.xml. Still finding my way through the test suite. Makes sense. Incorporated.